File: Algorithms/Sorting_Algorithms/Selection_Sort/selection_sort.rb
array = Array[4,456,45,657,34,8,987,1,78,50]

size = array.size

min = 0

for i in 0..size
    for j in (i+1)..size
        if array[j-1] < array[i-1]
            min = array[j-1]
            array[j-1] = array[i-1]
            array[i-1] = min
        end
    end
end

puts array

File: Algorithms/Sorting/Selection_Sort/SelectionSort.py
def selection_sort(lst):
    for i in range(len(lst) - 1):
        k = min(range(i, len(lst)), key=lst.__getitem__)
        lst[i], lst[k] = lst[k], lst[i]
    return lst
         
lst = [8,3,6,2]
print(selection_sort(lst))
